bin/templates/cordova/lib/builders/ProjectBuilder.js

```diff
@@ -37,23 +38,35 @@ const TEMPLATE =
 class ProjectBuilder {
     constructor (rootDirectory) {
         this.root = rootDirectory || path.resolve(__dirname, '../../..');
-        this.binDir = path.join(this.root, 'app', 'build', 'outputs', 'apk');
+        this.apkDir = path.join(this.root, 'app', 'build', 'outputs', 'apk');
+        this.aabDir = path.join(this.root, 'app', 'build', 'outputs', 'bundle');
     }
 
     getArgs (cmd, opts) {
-        if (cmd === 'release') {
-            cmd = 'cdvBuildRelease';
-        } else if (cmd === 'debug') {
-            cmd = 'cdvBuildDebug';
-        }
+        var args;
+        if (opts.packageType === PackageType.BUNDLE) {
+            if (cmd === 'release') {
+                cmd = ':app:bundleRelease';
+            } else if (cmd === 'debug') {
+                cmd = ':app:bundleDebug';
+            }
+
+            args = [cmd, '-b', path.join(this.root, 'build.gradle')];
+        } else {
+            if (cmd === 'release') {
```

Comment:
I have a pending commit to avoid overwriting the function argument `cmd`, but as per discussion at #786 I did not use ternary style condition statements.